portal-impl/test/unit/com/liferay/portal/dao/sql/transformer/BaseSQLTransformerLogicTestCase.java

```diff
@@ -240,7 +240,7 @@ protected String getCrossJoinTransformedSQL() {
 	}
 
 	protected String getDateFormatOriginalSQL() {
-		return "select foo from Foo where DATE2SEC(foo)";
+		return "select foo from Foo where TRUNCATE_TO_SECONDS(foo)";
```

Comment:
I think we should add a date of example, this command does not exist even if we ignore the TRUNCATE_TO_SECONDS :thinking: 

Something like:

`select foo from Foo where TRUNCATE_TO_SECONDS(foo) = '2024-10-04T09:03:00Z'`